dev.md

```diff
@@ -6,6 +6,23 @@ overview tbd.
 
 tbd.
 
+**Setup a local dev environment**
+
+You can use the docker-compose [stack](stack) to start all the necessary infrastructure components (like e.g. kafka and minio).
+
+**Applications and profiles**
+
+To start the application you can use the profiles:
```

Comment:
Which application(s)? Can you name them?

Reply:
Basically every service. I added 2 mermaid.js diagrams to understand the connection between our services.